src/App.tsx

```diff
@@ -9,19 +9,23 @@ function App() {
   return (
     <div className="container">
       <header className="header">
-        <div className="banner">
-          <div className="fancyText">Vi gifter oss</div>
-          <div className="fancyText large">Emil & Johan</div>
-          <div className="fancyText">16-17 augusti 2025</div>
+        <div className="imageBanner">
+          <h1 className="fancyText h1">Emil & Johan</h1>
+          <h2 className="fancyText h2">16-17 augusti 2025</h2>
+          <div className="buttonWrapper">
+            <Button title="o.s.a" wide />
```

Comment:
This adds an o.s.a button here, but there is one in Welcome.tsx too. Is that deliberate? Or should we remove one of them?